packages/gleam/0.27.0/run

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+shift
+cd project
+gleam run $(cat ../compile_args 2> /dev/null) -- "$@"
```

Comment:
Wondering if it makes sense to discard the three first lines of stdout since they don't matter when using piston

Reply:
Honestly I have not idea

Reply:
Im making a PR for it that can be merged if the gleam output seems weird